pkg/network/ebpf/c/protocols/http2/decoding-tls.h

```diff
@@ -438,7 +438,7 @@ static __always_inline void tls_fix_header_frame(tls_dispatcher_arguments_t *inf
 
 static __always_inline bool tls_get_first_frame(tls_dispatcher_arguments_t *info, frame_header_remainder_t *frame_state, http2_frame_t *current_frame, http2_telemetry_t *http2_tel) {
     // No state, try reading a frame.
-    if (frame_state == NULL) {
+    if (frame_state == NULL || (frame_state->remainder == 0 && frame_state->header_length == 0)) {
```

Comment:
modify the comment
open a jira ticket to investigate why do we have a frame state where both remainder and header_length are zero

Reply:
https://datadoghq.atlassian.net/browse/USMON-797